src/Native/System.Native/pal_networking.h

```diff
@@ -40,43 +44,65 @@ enum GetHostErrorCodes
     PAL_NO_ADDRESS = PAL_NO_DATA,
 };
 
-struct PalIpAddress
+/**
+ * Address families recognized by {Get,Set}AddressFamily
```

Comment:
Would be worth adding a comment that these values must be the same as those in System.Net.Sockets.AddressFamily.